front/src/actions/dashboard/boxes/devices.js

```diff
@@ -0,0 +1,164 @@
+import { RequestStatus } from '../../../utils/consts';
+import createBoxActions from '../boxActions';
+import createDeviceActions from '../../device';
+import update from 'immutability-helper';
+import get from 'get-value';
+import debounce from 'debounce';
+
+const BOX_KEY = 'Devices';
+
+function createActions(store) {
+  const boxActions = createBoxActions(store);
+  const deviceActions = createDeviceActions(store);
+
+  const actions = {
+    async getDevices(state, box, x, y) {
+      boxActions.updateBoxStatus(state, BOX_KEY, x, y, RequestStatus.Getting);
+      try {
+        const devices = await state.httpClient.get(`/api/v1/device`); //TODO Filter
```

Comment:
I don't think loading all devices in Gladys is wise, this request is going to be gigantic! Can you filter with only the selected devices?